src/frame/js/web-messenger.jsx

```diff
@@ -194,13 +194,13 @@ export function init(props = {}) {
                                     storage.removeItem(`${props.appId}.appUserId`);
                                     storage.removeItem(`${props.appId}.sessionToken`);
 
-                                    return store.dispatch(batchActions([
+                                    store.dispatch(batchActions([
```

Comment:
😕  why change this logic? This is a case where we recovered from bad `appUserId`/`sessionToken` and just booted as a fresh anonymous user instead. I think the init can be considered a success here